c/polyjuice.h

```diff
@@ -1010,6 +1018,70 @@ int handle_transfer(gw_context_t* ctx,
   return 0;
 }
 
+int handle_native_token_transfer(gw_context_t* ctx, uint32_t from_id,
+                                 uint256_t value, gw_reg_addr_t* from_addr) {
+  if (g_creator_account_id == UINT32_MAX) {
+    ckb_debug("[handle_native_token_transfer] g_creator_account_id wasn't set.");
+    return ERROR_NATIVE_TOKEN_TRANSFER;
+  }
```

Comment:
It would be better to check the `g_creator_account_id` in [`parse_args`]( https://github.com/nervosnetwork/godwoken-polyjuice/pull/173/files#diff-8a43ccc334a831d56b5bdfa4712228904f3823f6ec2b82941664ca849ab496b0R205-R207), before setting `g_eoa_transfer_flag = true`.

Reply:
g_creator_account_id and g_eoa_transfer_flag are set for good right now. I just add those checks to prevent handle_native_token_transfer gets used in other places improperly in the future.